ui/src/views/activities/SearchActivities.tsx

```diff
@@ -0,0 +1,63 @@
+import { Row, Col, Form, DatePicker, Button, Switch } from "antd"
+import moment, { Moment } from "moment"
+
+export interface SearchActivitiesValues {
+    period?: [Moment, Moment]
+    productionOnly?: boolean
+}
+
+export interface SearchActivitiesProps {
+    initialValues?: SearchActivitiesValues 
+    onClickSearch(values: SearchActivitiesValues): void
```

Comment:
It defines only the initial value and action for clicking the search button as a property of the component.